Library/Formula/chadwick.rb

```diff
@@ -0,0 +1,20 @@
+class Chadwick < Formula
+  desc "Chadwick tools for parsing Retrosheet MLB play-by-play files."
+  homepage "http://chadwick.sourceforge.net/doc/index.html"
+  url "https://downloads.sourceforge.net/project/chadwick/chadwick-0.6/chadwick-0.6.4/chadwick-0.6.4.tar.gz"
+  sha256 "2fcc44b4110c3aebf8b9f3daaccf22ccaba3760d3d878e65d38b86127b913559"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--disable-debug", "--disable-dependency-tracking"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
```

Comment:
Let's just test one of these, rather than all of them. Ideally the test should do something more than `--version` or `--help` as well, Thanks!